### PR TITLE
feat(service-portal): Signature Lists Collection

### PR DESCRIPTION
**Pull request by albinagu**
WIP: New module for signature collection, only initial setup, behind feature flag

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by thorkellmani — CHANGES_REQUESTED** [review]
Mostly looks good, some comments tho

File: libs/service-portal/signature-collection/codegen.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+schema:
+  - apps/api/src/api.graphql
+generates:
+  libs/service-portal/signature-collection/src/types/schema.ts:
+    plugins:
+      - typescript
+      - typescript-operations
+hooks:
+  afterAllFileWrite:
+    - prettier --write
```
[the remark this answers is not in the document]
Comment (albinagu) [reply libs/service-portal/signature-collection/codegen.yml:10]: there will be tho 🤔

File: libs/service-portal/signature-collection/project.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      "outputs": [
+        "{workspaceRoot}/coverage/libs/service-portal/signature-collection"
+      ]
+    },
+    "codegen/frontend-client": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "graphql-codegen --config libs/service-portal/signature-collection/codegen.yml"
+      }
+    },
```
[the remark this answers is not in the document]
Comment (albinagu) [reply libs/service-portal/signature-collection/project.json:31]: will be adding soon

File: libs/service-portal/signature-collection/src/lib/messages.ts [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+  /* File Upload */
+  uploadFile: {
+    id: 'sp.information:uploadFile',
+    defaultMessage: 'Skila einning meðmælum á blaði',
+    description: '',
+  },
+  uploadFileDescription: {
+    id: 'sp.information:uploadFileDescription#markdown',
+    defaultMessage:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin non felis augue. Integer erat sapien, auctor ac porttitor ut, lacinia quis erat. Vivamus quis sollicitudin turpis. Aliquam erat volutpat. Donec ut ante malesuada, varius erat ut, scelerisque sapien. Sed in eros at purus sollicitudin feugiat non non arcu. Nullam at sapien eu nisi tempor pulvinar ut sed nibh. Sed ex libero, vestibulum vitae eros sit amet, sagittis cursus lacus.',
```
[the remark this answers is not in the document]
Comment (albinagu) [reply libs/service-portal/signature-collection/src/lib/messages.ts:124]: yes, many texts that are still TBD

File: libs/service-portal/signature-collection/src/screens/ViewList/paperUpload.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            <Box marginY={5}>
+              <Text>{formatMessage(m.uploadFileDescription)}</Text>
+            </Box>
+            <InputFileUpload
+              fileList={[]}
+              header={formatMessage(m.uploadHeader)}
+              description={formatMessage(m.uploadText)}
+              buttonLabel={formatMessage(m.uploadButton)}
+              onChange={() => console.log('todo')}
+              onRemove={() => console.log('todo')}
```
[the remark this answers is not in the document]
Comment (albinagu) [reply libs/service-portal/signature-collection/src/screens/ViewList/paperUpload.tsx:50]: will be addressing in the next PR when we have the queries

**Review by thorkellmani — APPROVED** [review]
LGTM

**Review by lodmfjord — APPROVED** [review]
LGTM

**Pull request merged**